spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,6 +31,12 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:workdir) { "#{WORK_BASE}/workdir" }
+      let (:wixtestfiles) { "spec/fixtures/wix/resources/windows/wix" }
+      let (:include_filename) { "include/include-sample-1.wxs" }
```

Comment:
include_filename, ui_filename, bitmap_filename and generic_file are defined but never used.  They can be removed.